doc/articles/features/using-skia-rendering.md

```diff
@@ -0,0 +1,150 @@
+---
+uid: Uno.Skia.Rendering
+---
+
+# Using the Skia Rendering
+
+The Skia rendering engine is available for all platforms supported by Uno Platform. **Since Uno.Sdk 6.0+, it is the default rendering engine across all Uno Platform targets.** On **Windows (WinUI)**, **WebAssembly (Wasm)**, **Android**, and **iOS**, you can also choose to use the native rendering engine instead.
```

Comment:
```suggestion
The Skia rendering engine is available for all Uno Platform targets. **Since Uno.Sdk 6.0+, it is the default rendering engine.** On **WebAssembly (WASM)**, **Android**, and **iOS**, you can opt-in to use the native rendering engine instead. The **WinAppSDK** target is not provided by Uno Platform directly, so it only offers **native rendering mode**. 
```